src/coreclr/tests/src/Common/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+
+  <Import Project="..\Directory.Build.props" />
+
+  <PropertyGroup>
+    <UseLiveBuild>false</UseLiveBuild>
```

Comment:
Why is this?

Reply:
Do we want to build every tool in our repo against the live ref assemblies of libraries? I spoke with @jkoritzinsky about this and he suggested to build these common tools and the generated xunit wrappers against the sdk refs. I can add a comment.

Reply:
I thought that was the point of a combined repo, no? To build everything live?

Reply:
Oh this is just a tool, not tests.

Reply:
Yep. The assemblies in tests/src/Common other than CoreClrTestLibrary are tools to assist running the tests and are not part of the tests themselves.

Reply:
Makes sense.